step-2/application/src/modules/Commits/Actions.ts

```diff
@@ -44,4 +55,11 @@ export const fetchData = (dispatch: Dispatch) => {
     });
 };
 
+export const setSelectedCommitId = (dispatch: Dispatch, id: string) =>{
+  dispatch(setCommitIds(id));
+}
+
+
 export type DataActions = ReturnType<typeof fetchDataSuccess>;
+export type OtherActions = ReturnType<typeof setCommitIds>;
```

Comment:
you can combine these using union 

DataActions = ReturnType<typeof fethcDataSucces | typeof setCommitIds>
```suggestion
export type OtherActions = ReturnType<typeof setCommitIds>;
```